e2e/utils.go

```diff
@@ -563,6 +611,12 @@ func validatePVCClone(sourcePvcPath, clonePvcPath, clonePvcAppPath string, f *fr
 	// temporary clone+ total clones
 	totalCloneCount := totalCount + totalCount + 1
 	validateRBDImageCount(f, totalCloneCount)
+	if *pvc.Spec.VolumeMode == v1.PersistentVolumeFilesystem {
```

Comment:
why we need to delete the pod here?

Reply:
While calling writeDataInPod() function, internally the createApp function is being called which creates the pod but in the later stage the pod is not being deleted. And due to this I was getting failure with message "unable to delete PVC" and need to delete the pod first.